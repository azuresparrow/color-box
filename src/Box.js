import React from "react";

const Box = ({width = 10, height = 10, backgroundColor='black', id, handleRemove}) => {
    const removeRect = () => handleRemove(id);
    return (
    <div style={{
        id,
        height:`${height}em`,
        width:`${width}em`,
        backgroundColor
    }}>
        <button onClick={removeRect}>X</button>
    </div>);
}


export default Box;