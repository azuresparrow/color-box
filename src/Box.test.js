import { render } from '@testing-library/react';
import Box from './Box';

test('renders', () => {
  render(<Box />);
});

it('matches snapshot', () => {
  const { asFragment } = render(<Box/>);
  expect(asFragment()).toMatchSnapshot();
});