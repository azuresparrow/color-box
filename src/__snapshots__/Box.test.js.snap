// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    style="height: 10em; width: 10em; background-color: black;"
  >
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
